Cargo.lock

```diff
@@ -94,7 +94,7 @@ dependencies = [
  "tokio",
  "tokio-util 0.7.2",
  "tracing",
- "zstd",
+ "zstd 0.12.3+zstd.1.5.2",
```

Comment:
qq: why do we need two versions of zstd, 0.12.3 and 1.5.2? Is that how I read it?

Reply:
Not idea, Cargo.lock is automatically generated 🤷‍♂️ 
@nagisa can you please help here

Reply:
Don't think it matters too much. We good!

Reply:
The one version is the Rust crate, the other is the bundled libzstd. I would recommend upgrading straight to 0.13 series though…

Reply:
@shreyan-gupta also see [this zulip thread](https://near.zulipchat.com/#narrow/stream/300659-Rust-.F0.9F.A6.80/topic/.E2.9C.94.20Transitive.20dependency.20conflict)